src/wmnds/patterns/footer/_footer.js

```diff
@@ -0,0 +1,31 @@
+const footerJs = () => {
+  const mobileFooter = window.matchMedia('(max-width: 1024px)');
+  const collapseMenus = document.querySelectorAll('.wmnds-collapse-heading');
+  function handleMobileFooter(mq) {
```

Comment:
```suggestion
  const handleMobileFooter = (mq) => {
```

Use arrow syntax to be consistent with the handleToggle function and ES6 syntax being used.